README.md

```diff
@@ -424,7 +424,7 @@ To include token refreshing in your application you need to create a table to st
 Use below command to create a model `RefeshToken` with columns to store the token and the user reference
 
 ```bash
-$ rails generate model refresh_token token:string:uniq user:references
+$ rails generate model refresh_token token:string:uniq user:references expire_at:datetime
```

Comment:
Or for those already using refresh tokens:

`$ rails generate migration add_expire_at_to_refresh_token expire_at:datetime`

Reply:
Then we'll probably need to set the `expire_at` date for missing rows

Reply:
Good comments. Not sure it's really needed in the documentation (or would need to end up in an "Upgrade" section), as I think it's fairly self explanatory.